streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -890,6 +914,22 @@ public class StreamsConfig extends AbstractConfig {
                     in(AT_LEAST_ONCE, EXACTLY_ONCE, EXACTLY_ONCE_BETA, EXACTLY_ONCE_V2),
                     Importance.MEDIUM,
                     PROCESSING_GUARANTEE_DOC)
+            .define(RACK_AWARE_ASSIGNMENT_STRATEGY_CONFIG,
+                    Type.STRING,
+                    RACK_AWARE_ASSIGNMENT_STRATEGY_NONE,
+                    in(RACK_AWARE_ASSIGNMENT_STRATEGY_NONE, RACK_AWARE_ASSIGNMENT_STRATEGY_MIN_TRAFFIC),
+                    Importance.MEDIUM,
+                    RACK_AWARE_ASSIGNMENT_STRATEGY_DOC)
+            .define(RACK_AWARE_ASSIGNMENT_TRAFFIC_COST_CONFIG,
+                    Type.INT,
+                    null,
```

Comment:
Should we have any default value? If not, how can we give guidance to users how to set it and document it? If not, it would be very difficult for users to set.

Same for non-overlap cost below.

Reply:
The default value is different for `HAAssignor` and `StickyAssignor`, so the null here means use default in assignor. If it's set, it would override assignor's default. I guess we can guide to set relative values. Maybe we can mention default in HAAssignor is 10, 1 and in sticky assignor, it will be 1, 10...

Reply:
Thanks. We will need a docs PR anyway -- can you open one in the next days and include this information? -- Wondering if the `description` in `StreamsConfig` should also mention the different default values for both assignors and explain that `null` means use those defaults?